modules/ding_frontend/ding_frontend.menu_block.inc

```diff
@@ -28,21 +28,6 @@ function ding_frontend_menu_block_defaults() {
   $menu_block = new stdClass();
   $menu_block->disabled = FALSE; /* Edit this to true to make a default menu_block disabled initially */
   $menu_block->api_version = 1;
-  $menu_block->name = '2';
-  $menu_block->label = 'Secondary menu (menu block)';
-  $menu_block->description = '';
-  $menu_block->depth = 1;
-  $menu_block->expanded = 0;
-  $menu_block->follow = '0';
-  $menu_block->level = 1;
-  $menu_block->parent = 'menu-secondary-menu:0';
-  $menu_block->sort = 0;
-  $menu_block->title_link = 0;
-  $export['2'] = $menu_block;
-
-  $menu_block = new stdClass();
-  $menu_block->disabled = FALSE; /* Edit this to true to make a default menu_block disabled initially */
-  $menu_block->api_version = 1;
```

Comment:
Is it intentional that this menu is removed? It does not seem to be related to the change at hand.